index.html

```diff
@@ -9,46 +9,40 @@
 </head>
 
 <body>
```

Comment:
There is a problem here, I have the images/top-bg.svg set on the main as background. I can't debug this, because this page you printed is not the one that I just pushed in the last commit. Also, the h2 is white here, the only color added was is the h1.